### PR TITLE
chore: bump jsii version

### PR DESCRIPTION
**Pull request by iliapolo**
Needed to nuget publishing due to: https://github.com/aws/jsii/commit/9ef8f17e11fa3403043c3237463c16e08020c696

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/cdk8s-plus/.projenrc.js [hunk 1]
```diff
@@ -48,4 +48,10 @@ project.addScripts({
   build: 'jsii --silence-warnings=reserved-word && yarn docgen'
 });
 
+// this has to be done so because otherwise its gets overridden
+// by the jsii-release version configured in projen.
+project.addDevDependencies({
+  'jsii-release': Semver.caret('0.1.8'),
```
Comment (eladb) [inline packages/cdk8s-plus/.projenrc.js:54]: jsii-release is only needed in the root package.json, why do we need it here as well?

Comment (iliapolo) [reply packages/cdk8s-plus/.projenrc.js:54]: yeah you're right. I'll change it.

Comment (iliapolo) [reply packages/cdk8s-plus/.projenrc.js]: @eladb Done

**Comment by iliapolo** [comment]
Closing as this upgrade is currently not possible due to jsii version conflicts with constructs

**Pull request closed without merging**